packages/sdk-client-v3/src/middleware/create-concurrent-modification-middleware.ts

```diff
@@ -5,7 +5,13 @@ import {
   Next,
 } from '../types/types'
 
-export default function createConcurrentModificationMiddleware(): Middleware {
+export default function createConcurrentModificationMiddleware(
+  customLogic?: (
```

Comment:
I believe we can find a better name for this option, maybe something within the line `customResponseModifier` or simply `modifierFunction`